gr-utils/modtool/core/add.py

```diff
@@ -223,8 +236,7 @@ def _run_lib(self):
         if self.info['version'] in ('38', '310'):
             fname_h = self.info['blockname'] + '.h'
             fname_cc = self.info['blockname'] + '.cc'
-            if self.info['blocktype'] in ('source', 'sink', 'sync', 'decimator',
-                                          'interpolator', 'general', 'hier', 'tagged_stream'):
+            if self.info['blocktype'] not in ('noblock',):
```

Comment:
The right is the same as the left, but the left would have gotten even longer by adding `rfnoc_block` so I though this was neater.